### DISCUSSION TITLE
Dell U2720Q is not controlled by the app when in DCI-P3 colour mode

### DISCUSSION THREAD

**Discussion by tyger** [MonitorControl/MonitorControl#928] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

MonitorControl works fine with Dell U2720Q and capable of controlling its brightness when monitor is pretty much any colour mode except for DCI-P3.
When I switch to DCI-P3 in monitor OSD menu it changes brightness, it is still possible to controll brightness with monitor itself, but MonitorControl has no effect on it.

### Expected behavior

MonitorControl should be able to control brightness of the monitor when it is in DCI-P3 colour mode.

### Environment Information (please complete the following information)

```markdown
- macOS version: 12.2 Monterey
- Mac model: MacBook Pro (16-inch, 2021)
- MonitorControl version: v4.0.2
- Monitor(s): Dell U2720Q
- Monitor Cable(s)/Connection(s): USB-C to USB-C (came with monitor)
- Apple Silicon/M1 (yes or no): yes
```

**Comment by waydabber** [comment]
This seems to be a monitor specific issue, it seems like the display is not accepting DDC commands in this mode.

**Reply by tyger** [reply]
Thanks for answer.
I wonder is there an easy way to check it somehow? Could it be that monitor only ignore DDC commands for brightness control?

**Comment by mahad885** [comment]
I also have the same issue where I can no longer adjust the brigthness for my Dell monitor and the using the cursor method to change the brighness also does not work.

My monitor is a S2722QC
